### PR TITLE
Ignore print in scripts for generating fixtures

### PR DESCRIPTION
**Pull request by peterjc**
Pulled out of work on #111 to get this merged quickly to avoid more conflicts.

We may at some point adopt something like ``click`` for pretty command line output or logging, but most likely the fixture generating scripts will remain simple and self-contained and using ``print`` seems simple and natural here.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [ ] This is a documentation update

## Action Checklist

- [x] Work on a single issue/concept (if there are multiple separate issues to address, please use a separate pull request for each)
- [ ] Fork the `pyani-plus` repository under your own account (please [allow write access for repository maintainers](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
- [x] Set up an appropriate development environment (please see `CONTRIBUTING.md` for this repository)
- [x] Create a new branch with a short, descriptive name
- [x] Work on this branch
  - [x] style guidelines have been followed (please see `CONTRIBUTING.md` for this repository)
  - [ ] new code is commented, especially in hard-to-understand areas
  - [ ] corresponding changes to documentation have been made
  - [ ] tests for the change have been added that demonstrate the fix or feature works
- [x] Test locally with `pytest-plus -v` **non-passing code will not be merged**
- [x] Rebase against `origin/master`
- [x] Check changes with linting/formatting tools before submission (please see `CONTRIBUTING.md` for this repository)
- [x] Commit branch
- [x] Submit pull request, describing the content and intent of the pull request
- [x] Request a code review
- [x] Continue the discussion at [`Pull requests` section](https://github.com/widdowquinn/pyani-plus/pulls) in the `pyan-plus` repository

### CODE REVIEW

File: tests/generate_fixtures/generate_target_dnadiff_files.py [hunk 1]
```diff
@@ -48,7 +48,6 @@
 
 from pyani_plus.tools import (
     get_delta_filter,
-    get_dnadiff,
```
Comment (kiepczi) [inline tests/generate_fixtures/generate_target_dnadiff_files.py:51]: Is there a reason why `pre-commit-ci[bot]` doesn't like this import and keeps removing it?

Comment (peterjc) [reply tests/generate_fixtures/generate_target_dnadiff_files.py:51]: It looks like an unused import, so it should be removed.

Comment (peterjc) [reply tests/generate_fixtures/generate_target_dnadiff_files.py:51]: Ah - I see it now, I messed up the merge conflict and dropped the line using it...

**Comment by peterjc** [comment]
Rebased with an extra commit to actually use the dnadiff version in the generator script.

**Pull request merged**